app/assets/stylesheets/mobile/header.scss

```diff
@@ -163,6 +162,16 @@ header {
 }
 
 /* This class is added when the user open the drawer */
-#app.draw > * {
-          transform: translateX(-80%);
+.draw {
+    #drawer, #main_nav {
```

Comment:
Avoid using id selectors<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `main_nav` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own line